src/poliastro/maneuver.py

```diff
@@ -60,28 +56,101 @@ def __getitem__(self, key):
     def impulse(cls, dv):
         """Single impulse at current time.
 
+        Parameters
+        ----------
+        dv: np.array
+            Velocity components of the impulse.
         """
+
         return cls((0 * u.s, dv))
 
     @classmethod
     def hohmann(cls, orbit_i, r_f):
-        """Compute a Hohmann transfer between two circular orbits.
+        r"""Compute a Hohmann transfer between two circular orbits.
+
+        By defining the relationship between orbit radius:
+
+        .. math::
+            R = \frac{r_{f}}{r_{i}}
 
+        The Hohmann maneuver velocities can be expressed as:
+
+        .. math::
+            \begin{align}
+                \Delta v_{a} &= v_{i}\left ( \sqrt{\frac{2R}{1+R}} - 1\right )\\
+                \Delta v_{b} &= \frac{v_{i}}{\sqrt{R}}\left (\sqrt{\frac{2}{1+R}} - 1 \right )\\
+            \end{align}
+
+        The time that takes to compelte the maneuver can be computed as:
+
+        .. math::
+            \tau_{trans} = \pi \sqrt{\frac{(r{i} + r{f})^{3}}{8\mu}}
+
+        Parameters
+        ----------
+        orbit_i: poliastro.twobody.orbit.Orbit
+            Initial orbit
+        r_f: astropy.unit.Quantity
+            Final altitude of the orbit
         """
-        # TODO: Check if circular?
-        r_i = orbit_i.a
-        v_i = orbit_i.v
-        k = orbit_i.attractor.k
-        R = r_f / r_i
-        dv_a = ((np.sqrt(2 * R / (1 + R)) - 1) * v_i).decompose()
-        dv_b = (-(1 - np.sqrt(2 / (1 + R))) / np.sqrt(R) * v_i).decompose()
-        t_trans = (np.pi * np.sqrt((r_i * (1 + R) / 2) ** 3 / k)).decompose()
-        return cls((0 * u.s, dv_a), (t_trans, dv_b))
+
+        if orbit_i.ecc == 0:
```

Comment:
What happens with orbits that have an eccentricity of 1e-5? We could either:

* Accept some tolerance
* Provide a "circularize" maneuver

Reply:
Sure! I will add the variable for `tolerance`.  When you say 'circularize' you mean to return a `Maneuver` object that:
1. Changes `Orbit.ecc` to zero
2. After `ecc` correction, applies Hohmann maneuver

Reply:
I would say that circularize would:

1. Propagate until pericenter (or apocenter?)
2. Apply tangential burn

And then we would combine it with the Hohmann transfer. How does that sound?